src/checkout-buttons/create-checkout-button-registry.ts

```diff
@@ -246,7 +253,13 @@ export default function createCheckoutButtonRegistry(
             store,
             checkoutActionCreator,
             formPoster,
-            paypalCommercePaymentProcessor
+            paypalCommercePaymentProcessor,
+            orderActionCreator,
+            countryActionCreator,
+            new ConsignmentActionCreator(
```

Comment:
(optional) It might be better to create a variable for ConsignmentActionCreator like countryActionCreator

Reply:
fixed